Makefile

```diff
@@ -2,6 +2,7 @@ BUILD_DIR=$(shell pwd)/build
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
 CERT_GEN=$(shell base64 -w 0 scripts/cert-gen.sh)
 VERSION=1.0.0-alpha.1
+TODAY=$(shell date +"%d-%m-%Y-%H-%M-%S")
```

Comment:
+1 to the idea of getting the `build_id` from environment if available.  
@praveenkumar @hferentschik I do not have strong opinion about using the shasum. So it is up to you. Th idea behind `build_id` is to identify each ISO with the build_id and track it. So as long as we are able to do that I am ok with it.